src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -90,7 +90,7 @@ internal partial class SuggestedActionsSourceProvider : ISuggestedActionsSourceP
             if (textBuffer.IsInLspEditorContext())
                 return null;
 
-            var asyncEnabled = _globalOptions.GetOption(SuggestionsOptions.Asynchronous) ?? _globalOptions.GetOption(SuggestionsOptions.AsynchronousFeatureFlag);
+            var asyncEnabled = _globalOptions.GetOption(SuggestionsOptions.Asynchronous) ?? !_globalOptions.GetOption(SuggestionsOptions.AsynchronousQuickActionsDisableFeatureFlag);
```

Comment:
no.  this isn't right.  it should be an &&, not an ?? in main and main-vs-deps.

Reply:
Wired.. It sounds like we are missing some commits in main?